prow/test/integration/test/sinker_test.go

```diff
@@ -173,9 +305,27 @@ func TestDeletePod(t *testing.T) {
 			if err != nil {
 				t.Fatal(err)
 			}
-			t.Logf("Pod %s exist: %v", tt.name, exist)
-			if want, got := tt.wantDelete, !exist; want != got {
-				t.Fatalf("Want deleted: %v. Got deleted: %v", want, got)
+			t.Logf("Pod %s exist: %v", pod.Name, exist)
+			if want, got := tt.wantPodDeleted, !exist; want != got {
+				t.Fatalf("Want pod deleted: %v. Got deleted: %v", want, got)
+			}
+
+			// Check for prowjob deletion.
+			if prowjob != nil {
+				exist = false
+				pjs := &prowjobv1.ProwJobList{}
+				err = kubeClient.List(ctx, pjs, ctrlruntimeclient.InNamespace(defaultNamespace))
+				if err != nil {
+					log.Fatalf("Failed listing prowjobs: %v", err)
```

Comment:
/hold

Reply:
Sorry, missed this, this needs to go in favor of `t.Fatalf`. Easiest to avoid this is to not import the `log` package :upside_down_face:

Reply:
fixed

Reply:
/hold cancel